api/src/test/java/com/valtech/aem/saas/api/indexing/dto/IndexContentPayloadDTOTest.java

```diff
@@ -0,0 +1,31 @@
+package com.valtech.aem.saas.api.indexing.dto;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.hamcrest.core.IsInstanceOf;
+import org.junit.jupiter.api.Test;
+
+class IndexContentPayloadDTOTest {
+
+  @Test
+  void testIndexContentPayloadBuild() {
+    assertThat(new IndexContentPayloadDTO("adventures content that is pushed.",
+        "WKND Adventures",
+        "https://wknd.site/us/en/adventures.html",
+        "/content/wknd/(?!www)",
```

Comment:
- [ ] wait, this is a regex? Can you explain please? The DTO's constructor is called repositoryPath and not repositoryPathRegex or something

Reply:
The field is named according spec -> https://decks.valtech.swiss/display/ICSAAS/Index+trigger+APIv3
The value is similar to a value that is used in the postman example requests that I got.

Reply:
mmh that it could be a regex was never communicated to me. let's clarify with saas-team

Reply:
- [ ] as discussed, please update test